README.md

```diff
@@ -61,4 +61,28 @@ $ make run
 ```
 
 You should be able to test the forum at [http://localhost:8090/dev/](http://localhost:8090/dev/).
-This is the development view that emulates a LTI consumer.
\ No newline at end of file
+This is the development view that emulates a LTI consumer.
+
+## Backoffice
+
+You can access the backoffice of the forum at [http://localhost:8090/admin/](http://localhost:8090/admin/)
+
+You first need to create a superuser account. 
+
+```bash
+$ bin/superuser
```

Comment:
Interesting to see there is no make rule for this task. In all django projects we have a `superuser` task is responsible of this. But maybe we already discuss of this point with @madmatah ?

Reply:
We've actually noticed it and as it's my first PR let it this way, to see what could come out of it :-)

Reply:
There was a time when I had a preference for the `bin/*` utilities rather than the make targets ;)
But I agree with you @lunika , to be consistent we should add a `superuser` target in the Makefile and remove this script, in another PR.

Reply:
The task has been added !